doc/configuration/tokens/yubikey.rst

```diff
@@ -30,4 +25,89 @@ Run the command like this::
    privacyidea -U https://your.privacyidea.server -a admin token \
    yubikey_mass_enroll --yubimode YUBICO
 
+This command initializes the token and stores the AES secret and prefix
+in privacyidea, so the toen is immediatly useful. You can choose the slot
+with ``--yubislot`` For further help call
+``privcyidea yubikey_mass_enroll`` with the ``--help`` option.
+
+The second way to enroll a yubikey token is also using ``yubikey_mass_enroll``,
+but with the option ``--filename`` to write to token configuration into the
+specified file.  The resulting file can then be imported into privacyidea:
+Select Tokens -> Import Tokens, Select "OATH CSV" and the file you just created.
+
+Third and last you can use this dialog to enroll a Yubikey AES mode token, if you have
+initialized the yubikey with the external *ykpersonalize* tool.
+
+.. figure:: images/enroll_yubikey.png
+   :width: 500
+
+   *Enroll a Yubikey AES mode token*
+
+
+Remarks about using Yubikey token
+.................................
+
+You can use Yubikey tokens for two more or less distinct applications. The first
+is using privacyideas PAM module. In this case privacyidea handles the policies
+for user access and password validation. This works fine, when you only use
+privacyidea for token validation.
+
+The second mode is using the standard PAM module for yubikeys from Yubico 
+``pam_yubico`` to handle the token validation. The upside ist that you can
+use the PAM module included with you distribution, but there are downsides as
+well.
+
+* You can't set a token PIN in privacyidea, because ``pam_yubico`` tries to
+  use the token PIN entered by the user as a system password (which is likely
+  to fail).
```

Comment:
i.e. the PIN will be stripped by pam_yubico and will not reach the privacyIDEA system...